_content/monographs/the-paccekabuddha-buddhist-ascetic_kloppenborg.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "The Paccekabuddha:
+A Buddhist Ascetic. A Study of the Concept of the 
+Paccekabuddha in Pali Canonical and Commentarial Literature"
+authors:
+  - "Ria Kloppenborg"
+external_url: "https://www.bps.lk/olib/wh/wh305_Kloppenborg_Paccekabuddha-Concept-In-Pali-Canon-Commentary.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1UjtEVCir4HE8CdOx3Zrv1HvVinVuIQZa/view?usp=drive_link"
+  - "https://drive.google.com/file/d/1orAgVjHpl64YY0z7WPaIrNY1Wsnrcc2g/view?usp=drive_link"
+course: cosmology
+tags:
+  - meditation
+  - asceticism
+  - renunciation
+  - solitude
```

Comment:
not a tag and redundant with `renunciation`

Perhaps we can add `interfaith` here instead as this is a bit of an example of the Buddha being ecumenical?